src/gdpr-banner/gdpr-banner.scss

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../dialog/index';
+
+@mixin iui-gdpr-banner {
+  @include iui-dialog;
+  border-radius: 0;
+  box-shadow: none;
+  background-color: var(--iui-color-background-3);
```

Comment:
Are we sure about this color? Looks inconsistent and goes against our documented use case which is _"Disabled components and headers of surfaces"_

Reply:
Fixed

Reply:
I see you are using `background-1` now. The `iui-dialog` mixin already includes that, so you don't need to repeat it.